src/components/typography/Typography.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Typography } from './Typography';
+
+export default {
+  title: 'Components/Typography',
+  component: Typography,
+  argTypes: {},
```

Comment:
Carlos,  it is neccesary when it's void?

Reply:
this is for extra arguments